organisms/organisation-unit-tree/new-features/select-by-group-level.feature

```diff
@@ -0,0 +1,56 @@
+# Generic
+
+Feature: Selection of descendant levels and/or groups inside an org unit
+
+  Background:
+    Given an org unit tree with root org unit 'Sierra Leone'
+    And descendants inside 'Sierra Leone' with multiple groups and levels
+
+  Scenario: Selection of a level or group inside an org unit
+    Given the user has opened the context menu 'Sierra Leone'
+    When the user selects a level or group type from the 'Select by >' menu
+    Then all org units of that level or group are selected as a single selection
+    And 'Sierra Leone' org unit visually indicates the selection made
+
+  Scenario: Hide non-available groups and levels in 'Select by' menus
+    Given the user has opened the context menu for 'Sierra Leone'
+    Then only groups that are present in descendants should appear in 'Select by group >' sub-menu
```

Comment:
I suppose we can add a second sentence here:
`And only groups that the user is authorized to select should appear in 'Select by group >' sub-menu`

I'm not 100% sure how the org unit tree works to this level of detail, but I can imagine that the user will have different authorities for different nodes (read and read/write), so that some nodes might be visible but not select-able?

Reply:
Yes, I think you are right. There isn't an example...yet. But I could imagine a case where a user can read the nodes but cannot make the selection. Added this line.